Documentation/platforms/arm/rp2040/boards/adafruit-feather-rp2040/index.rst

```diff
@@ -2,6 +2,8 @@
 Adafruit Feather RP2040
 =======================
 
+.. tags:: chip:rp2040
```

Comment:
what is the purpose of `chip` tag? doesn't the documentation structure itself make it obvious what are supported boards for a given chip? or is this a prerequisite to adding tag filtration in the future? (then its OK)